src/auctionManager.js

```diff
@@ -100,6 +100,10 @@ export function newAuctionManager() {
     }
   }
 
+  auctionManager.getLastAuctionId = function() {
+    return _auctions && _auctions.length && _auctions[_auctions.length - 1].getAuctionId()
```

Comment:
`_auctions` should always be an Array (it's initialized as an array (with `let`, might want to switch to `const` to make sure that's always the case)), so I think all these checks are unnecessary.

Reply:
made the change thanks